src/src/cron/prepareCertificateHtml.cron.ts

```diff
@@ -35,6 +35,7 @@ export class PrepareCertificateHtmlCron {
 				),
 			),
 		);
+		console.log('userForIssueCertificate list', userForIssueCertificate);
```

Comment:
remove console logs pls

Reply:
console log on dev is required for testing and track cron job flow. after testing complete on dev server then i will remove all logs and sent PR again